core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -477,6 +477,7 @@ public static boolean shouldPauseForFocusLoss() {
      */
     public static long getUpdateInterval() {
         String updateInterval = prefs.getString(PREF_UPDATE_INTERVAL, "0");
+        Log.e("UPREF/UpdateInterval" , " /Shared Preferences: " + updateInterval);
```

Comment:
If you are done with testing, please remove those logs again. That makes the code changes more "local".